PropertyControlStrings can't be resolved for PropertyFieldDateTimePicker control.

Thank you for reporting an issue, suggesting an enhancement, or asking a question. We appreciate your feedback - to help the team understand your
needs please complete the below template to ensure we have the details to help. Thanks!

**Please check out the [documentation](https://github.com/SharePoint/sp-dev-fx-property-controls/wiki) to see if your question is already addressed there. This will help us ensure our documentation is up to date.**

#### Category
[ ] Enhancement

[x] Bug

[ ] Question

#### Version

Please specify what version of the library you are using: [  Beta.1  ]

#### Expected / Desired Behavior / Question
Solution should build and run in the workbench.

#### Observed Behavior
Get multiple errors during build. Errors are that PropertyControlStrings can't be resolved.
I think this is used in multiple files in the package.
Example:
`./node_modules/@pnp/spfx-property-controls/lib/propertyFields/dateTimePicker/PropertyFieldDateTimePickerHost.js
Module not found: Error: Can't resolve 'PropertyControlStrings' in '/mnt/c/Dev/sp/lrn/spfx/react00/node_modules/@pnp/spfx-property-controls/lib/propertyFields/dateTimePicker`

#### Steps to Reproduce
Install packages
Add Property Pane field
gulp serve


Thanks!

--- a/src/propertyFields/collectionData/ICustomCollectionField.ts
+++ b/src/propertyFields/collectionData/ICustomCollectionField.ts
@@ -51,7 +51,7 @@ export interface ICustomCollectionField {
   /**
    * Custom field rendering support
    */
-  onCustomRender?: (field: ICustomCollectionField, value: any, onUpdate: (fieldId: string, value: any) => void) => JSX.Element;
+  onCustomRender?: (field: ICustomCollectionField, value: any, onUpdate: (fieldId: string, value: any) => void, rowUniqueId: string) => JSX.Element;
 }
 
 export enum CustomCollectionFieldType {

--- a/src/propertyFields/collectionData/PropertyFieldCollectionDataHost.module.scss
+++ b/src/propertyFields/collectionData/PropertyFieldCollectionDataHost.module.scss
@@ -124,6 +124,7 @@
 .tableCell {
   display: table-cell;
   padding: 0 10px;
+  vertical-align: middle;
 
   > div {
     margin-bottom: 8px;

--- a/src/propertyFields/collectionData/collectionDataItem/CollectionDataItem.tsx
+++ b/src/propertyFields/collectionData/collectionDataItem/CollectionDataItem.tsx
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import * as uuid from "uuid";
 import styles from '../PropertyFieldCollectionDataHost.module.scss';
 import { ICollectionDataItemProps, ICollectionDataItemState } from '.';
 import { TextField } from 'office-ui-fabric-react/lib/components/TextField';
@@ -23,6 +24,8 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
 
     // Create an empty item with all properties
     this.emptyItem = {};
+    this.emptyItem.uniqueId = uuid();
+
     for (const field of this.props.fields) {
       // Assign default value or null to the emptyItem
       this.emptyItem[field.id] = field.defaultValue || null;
@@ -149,10 +152,6 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
           this.checkAnyFieldContainsValue(crntItem) &&
           this.checkAllFieldsAreValid()) {
         this.props.fAddItem(crntItem);
-        // Clear all field values
-        this.setState({
-          crntItem: {...this.emptyItem}
-        });
       }
     }
   }

--- a/src/propertyFields/collectionData/collectionDataItem/CollectionDataItem.tsx
+++ b/src/propertyFields/collectionData/collectionDataItem/CollectionDataItem.tsx
@@ -23,16 +23,10 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
     super(props);
 
     // Create an empty item with all properties
-    this.emptyItem = {};
-    this.emptyItem.uniqueId = uuid();
-
-    for (const field of this.props.fields) {
-      // Assign default value or null to the emptyItem
-      this.emptyItem[field.id] = field.defaultValue || null;
-    }
+    let emptyItem = this.generateEmptyItem();
 
     this.state = {
-      crntItem: clone(this.props.item) || {...this.emptyItem},
+      crntItem: clone(this.props.item) || {...emptyItem},
       errorMsgs: [],
       showCallout: false
     };
@@ -152,6 +146,11 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
           this.checkAnyFieldContainsValue(crntItem) &&
           this.checkAllFieldsAreValid()) {
         this.props.fAddItem(crntItem);
+        // Clear all field values
+        let emptyItem = this.generateEmptyItem();
+        this.setState({
+          crntItem: {...emptyItem}
+        });
       }
     }
   }
@@ -356,7 +355,7 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
                           onGetErrorMessage={async (value: string) => this.urlFieldValidation(field, value, item)} />;
       case CustomCollectionFieldType.custom:
           if (field.onCustomRender) {
-            return field.onCustomRender(field, item[field.id], this.onValueChanged);
+            return field.onCustomRender(field, item[field.id], this.onValueChanged, item.uniqueId);
           }
           return null;
       case CustomCollectionFieldType.string:
@@ -384,6 +383,21 @@ export class CollectionDataItem extends React.Component<ICollectionDataItemProps
       });
     }
     return opts;
+  }
+
+   /**
+   * Creates an empty item with a unique id
+   */
+  private generateEmptyItem(): any {
+    // Create an empty item with all properties
+    let emptyItem:any = {};
+    emptyItem.uniqueId = uuid();
+    
+    for (const field of this.props.fields) {
+      // Assign default value or null to the emptyItem
+      emptyItem[field.id] = field.defaultValue || null;
+    }
+    return emptyItem;
   }
 
   /**

--- a/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
+++ b/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
@@ -216,7 +216,7 @@ export class CollectionDataViewer extends React.Component<ICollectionDataViewerP
           {
             (this.state.crntItems && this.state.crntItems.length > 0) && (
               this.state.crntItems.map((item, idx, allItems) => (
-                <CollectionDataItem key={idx}
+                <CollectionDataItem key={item.uniqueId}
                                     fields={this.props.fields}
                                     index={idx}
                                     item={item}
@@ -232,7 +232,7 @@ export class CollectionDataViewer extends React.Component<ICollectionDataViewerP
           <CollectionDataItem fields={this.props.fields}
                               index={null}
                               item={null}
-                              key={ this.state.crntItems ? this.state.crntItems.length + 1 : 1 }
+                              key={ this.state.crntItems ? this.state.crntItems.length + 1 : 1 } // This makes sure the empty row is rerendered from scratch when a new row is added / removed
                               sortingEnabled={this.props.enableSorting}
                               totalItems={null}
                               fAddItem={this.addItem}

--- a/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
+++ b/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
@@ -232,7 +232,6 @@ export class CollectionDataViewer extends React.Component<ICollectionDataViewerP
           <CollectionDataItem fields={this.props.fields}
                               index={null}
                               item={null}
-                              key={ this.state.crntItems ? this.state.crntItems.length + 1 : 1 } // This makes sure the empty row is rerendered from scratch when a new row is added / removed
                               sortingEnabled={this.props.enableSorting}
                               totalItems={null}
                               fAddItem={this.addItem}

--- a/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
+++ b/src/propertyFields/collectionData/collectionDataViewer/CollectionDataViewer.tsx
@@ -232,6 +232,7 @@ export class CollectionDataViewer extends React.Component<ICollectionDataViewerP
           <CollectionDataItem fields={this.props.fields}
                               index={null}
                               item={null}
+                              key={ this.state.crntItems ? this.state.crntItems.length + 1 : 1 }
                               sortingEnabled={this.props.enableSorting}
                               totalItems={null}
                               fAddItem={this.addItem}

